### DISCUSSION TITLE
Is it safe to add `@fluentui/react-jsx-runtime` as dev dependency?

### DISCUSSION THREAD

**Discussion by AkiraVoid** [microsoft/fluentui#29589] [Q&A] [answered]
It seems it's necessary to reference `@fluentui/react-jsx-runtime` package for JSX pragma if we use new slot API to build our component. I am wondering do we really need install this package even in production build. Since in production build, all JSX elements should have been compiled to vanilla JS code.

**Comment by layershifter** [comment, marked as answer]
Hey,

Yes, you need this package as production dependency as it will be used in your output 🐱 Here is [an example](https://play.swc.rs/?version=1.3.94-nightly-20231016.1&code=H4sIAAAAAAAAAy3LQQoCMQwF0H1P8ZdaGHIBkXHpVk9QagYiNi0xEWHw7s5i1o9HOWN%2Bvr%2B3UJfGKOG9FZeKTIl2vLbRze89rDLm5RWsHkLGpfq0%2BWT73k5aQqtLV1zGOByxJsDYwxSnh3xA5%2FT7A%2FNjTcF1AAAA&config=H4sIAAAAAAAAA1VQu27DMAzc8xUGkbGwmw4FmrVTh3TrB7AKnSg1JYGk0RqB%2F72WX3A28e54d%2BJ9VxSwV3clRjgWcDVLeqyqm8ZQTrBaFCqjXCr9deLgKa%2Fc1A3y%2B%2FAchoSiJOs8INoFw79saF0ideKTjYsja5opk5ZW6NwFZO8%2BOEWxmRy5fpKACQato%2FA2RgidbYAMtcE8U47G1iKjeQcz3T84fqPSlzRZWVawxKBcKFsC6cvz4W3Bf4jSe4Oqn8ikD%2B2BffB1t%2B3lIichzboaG51%2FspuzwespntuGNjbACzLfNN9tqvEK6%2BKaNLn2%2Fy88py2%2BAQAA).

**Reply by AkiraVoid** [reply]
I see. Thanks for your example and explanation.
